app/services/ssm_parameters.rb

```diff
@@ -18,6 +18,12 @@ def delete_parameter
                              })
   end
 
+  def get_parameters(ssm_paths)
+    client.get_parameters({
+                            names: ssm_paths
+                          })
+  end
```

Comment:
This is too raw. Please make this method private and write a method that returns a data type that is independent of the data returned by the AWS SDK.